test-harness/src/main/java/edu/umd/cs/findbugs/test/AnalysisRunner.java

```diff
@@ -133,17 +136,27 @@ public BugCollectionBugReporter run(Path... files) {
      * Create a jar file which contains all resource files. This is necessary to
      * let {@link Plugin#loadCustomPlugin(File, Project)} load custom plugin to
      * test.
-     * 
+     *
      * @return a {@link File} instance which represent generated jar file
      * @throws IOException
      * @throws URISyntaxException
      */
     private static File createTempJar() throws IOException, URISyntaxException {
         ClassLoader cl = AnalysisRunner.class.getClassLoader();
+
+        URL resource = cl.getResource("findbugs.xml");
+        URI uri = resource.toURI();
```

Comment:
actually a resource might be null here, but I think it's ok to fall with NPE in tests when test configuration is unexpected